skimage/color/colorconv.py

```diff
@@ -1077,6 +1084,8 @@ def lab2xyz(lab, illuminant="D65", observer="2", *, channel_axis=-1):
         warn(f'Color data out of range: Z < 0 in {invalid[0].size} pixels',
              stacklevel=3)
         z[invalid] = 0
+    else:
+        invalid = (np.array([], dtype=int))
```

Comment:
```suggestion
        invalid = np.array([], dtype=int)
```

are these extra parenthesis helpful? I feel like they act as no-ops here.